pkgs/by-name/lu/lunarvim/package.nix

```diff
@@ -0,0 +1,93 @@
+{ fd
+, fetchFromGitHub
+, fzf
+, git
+, lib
+, makeWrapper
+, neovim
+, nodePackages
+, python311
+, ripgrep
+, stdenv
+, tree-sitter
+}:
+let
+  lunar-python = python311.withPackages ( p: with p; [ pynvim ] );
+  runtimeDeps = [
+    fd
+    fzf
+    git
+    neovim
+    ripgrep
+    lunar-python
+    tree-sitter
+  ] ++ (with nodePackages; [
+    neovim
+  ]);
+in
+stdenv.mkDerivation rec {
```

Comment:
Instead of using rec, I think it is preferred to do it via a function as is documented [here](https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes)

Reply:
I had no idea :thinking: Well, it's not clear to me how to use finalAttrs (still a noob), so I opted with just writing out the string and removing `rec`. Hopefully that's sufficient.